### DISCUSSION TITLE
People email attribute

### DISCUSSION THREAD

**Discussion by vizsnyai** [microsoft-search/pnp-modern-search#4127] [Q&A]
Hi,
How can I print out the email address of a given user? It is important that it is not the UPN, but the mail attribute.
The users are synchronized from Active Directory and there are many mail users that I should be able to show in the phone book created in PnP Search. 
The users are visible in Entra, and there the mail value is the given eg. @gmail.com address, I should print this out.

Thanks

**Comment by vishalgoyal2472** [comment]
where exactly you want to show email address ?

**Comment by kasperbolarsen** [comment]
You have to provide some more info in order for us to provide an answer. Which data source are you using (SP/MS) and which layout do you intend to use?
